src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -280,26 +280,27 @@ Path createProjectForLinkedEdit(SourceSet ss, RunContext ctx, boolean reload) th
 		Util.mkdirs(srcDir);
 
 		Path srcFile = srcDir.resolve(name);
-		Util.createLink(srcFile, originalFile.toPath());
+		Util.createLink(srcFile, originalFile);
 
-		for (Source source : ss.getSources()) {
+		for (ResourceRef sourceRef : ss.getSources()) {
```

Comment:
sources are now resourceRefs ?

Reply:
Yes, "sources" (as in //SOURCES or --sources) are now being stored as simple ResourceRefs, just like the "resources" (//FILES and --files) are FileRefs (which are a ResourceRef + an optional alias). Before they were full-blown `Source` instances which a) are not always necessary so you're doing all the parsing for nothing and b) it's not future-proof with the idea of us wanting to create new project types.